### ISSUE TITLE
Doesn't show anything and shows no error

### ISSUE THREAD

**Issue by tillhanke** [thielepaul/photoprism-mobile#82]
Hi there, I just found the photoprism project and am really excited! 
I was looking for a nice photo mediathek solution for a while now and this looks really promising.

I got a simple docker version running on my raspi now for testing purposes and wanted to check out the android app.
But sadly it doesnt show anything. If I use the raspi ip with the default port it does not show any photos. And even the demo server is not loading correctly. It shows grey rectangles with ! inside them and nothing more. 

No error no nothing.
How could I go about finding out whats wrong here?

Thanks for your work!

**Comment by Jumn** [comment]
Same problem for me, 
the web view works fine, but the mobile app doesn't display any photos

![Screenshot_20210105-123127](https://user-images.githubusercontent.com/13217431/103643547-2e262b80-4f55-11eb-8930-23dc1b5daaef.jpg)

**Comment by imjoep** [comment]
I opened the same issue on the photoprism side (https://github.com/photoprism/photoprism/issues/818).  They said it's because 

> "recently improved the security of our API by using a random preview token by default - it previously was just public, so that you could theoretically find thumbs by knowing their sha1 hash."

Said you can downgrade photoprism until the issue is resolved

**Comment by alanpt** [comment]
Same problem here. Do you know what we might lose by downgrading? And the process of downgrading photoprism in a docker?

**Comment by Grumvar** [comment]
No need to downgrade, you can define the preview token in the photoprism configuration. The mobile app currently sends "public" as the token, if you define the following in the photoprism configuration it should work: PHOTOPRISM_PREVIEW_TOKEN: "public"

**Warning:** This disables the new security feature, therefore you need to remove that configuration when the photoprism app is updated.

**Comment by tillhanke** [comment]
I still cant see any previews from my server. Its just a blank page, not even the exclamation mark indicators...
![Screenshot_20210106-115108](https://user-images.githubusercontent.com/11048183/103760585-963c4680-5015-11eb-8b27-dbdf6879711a.png)

The raspi is just running in my local network under the ip 192.168.4.118
![Screenshot_20210106-115331](https://user-images.githubusercontent.com/11048183/103760785-e7e4d100-5015-11eb-9346-24b4732b68e9.png)

**Comment by fingolfin** [comment]
Having the same in the iOS testflight beta app.

**Comment by jakobroehrl** [comment]
Same Problem, but if I share a picture I can see it

**Comment by thielepaul** [comment]
I pushed an update that hopefully fixes this issue (or at least the one with the previewToken)

https://github.com/photoprism/photoprism-mobile#installation

**Comment by starkayc** [comment]
> I still cant see any previews from my server. Its just a blank page, not even the exclamation mark indicators...
> ![Screenshot_20210106-115108](https://user-images.githubusercontent.com/11048183/103760585-963c4680-5015-11eb-8b27-dbdf6879711a.png)
> 
> The raspi is just running in my local network under the ip 192.168.4.118
> ![Screenshot_20210106-115331](https://user-images.githubusercontent.com/11048183/103760785-e7e4d100-5015-11eb-9346-24b4732b68e9.png)

Not sure if you were able to fix it already but I had the same issue but I was able to get it to load by adding http:// in front of the ip so http://192.168.1.19:2342 for me worked instead of 192.168.1.19:2342 . Hope that helped out!

**Comment by ThinkDigitalSoftware** [comment]
So happy that this is getting active development. Thanks for the fix, @starkayc. That solved my issue. We need to put a confirmation screen with a check to make sure the server URL is valid. We also need an error message on the main screen

**Comment by jakobroehrl** [comment]
Photos are shown for me with the latest app and the demo server.
But I don't see any videos

**Comment by skippa** [comment]
Same issue, I've put my server IP with and without the http and port - nothing working. Server works because I can access via browser...

**Comment by esturniolo** [comment]
Same here. 
I've my server in the cloud and i tried the config with and without the https with no luck. 
Complete blank screen.

**Comment by skippa** [comment]
Managed to get it to work - I used http and the port number eg http://xxx.xxx.xxx.xxx:2342 and selected Photoprism only under authentication. It's clearly an alpha version as it's got bugs, very promising though!

**Comment by step21** [comment]
> Managed to get it to work - I used http and the port number eg http://xxx.xxx.xxx.xxx:2342 and selected Photoprism only under authentication. It's clearly an alpha version as it's got bugs, very promising though!

Yeah, same. Though at least an error would be nice on not connecting. Or a messsage in logs.

**Comment by nevado** [comment]
In case this helps anyone else, I just realised why mine wasn't working... you must also remove any trailing slash from the URL, i.e. http://xxx.xxx.xxx.xxx:2342 instead of http://xxx.xxx.xxx.xxx:2342/
